### PR TITLE
Simplified changelog

### PR DESCRIPTION
**Pull request by Titan-C**
This addresses #67 writing a chagelog to be show in sphinx documentation plus a hyperlink to this file in the main repository to avoid https://github.com/sphinx-gallery/sphinx-gallery/issues/67#issuecomment-151819973

### CODE REVIEW

**Comment by agramfort** [comment]
LGTM

**Comment by lesteve** [comment]
I am not too sure whether symbolic links in git repo work correctly under Windows. Maybe we don't really care that much since this is just used for the doc generation.

Still I would have the "real" changes.rst at the root of the project and the symbolic link in the `doc` folder. This way changes.rst can be viewed correctly on github. I think using `.. include ../changes.rst` in doc/changes.rst works and I reckon this is simpler than the symbolic link.

Slight niggle, maybe use capitals i.e. CHANGES.rst. This is what joblib does and seems to be some kind of convention. For example see https://en.wikipedia.org/wiki/README.

**Comment by Titan-C** [comment]
Last PR before release of Sphinx-Gallery v0.0.12

File: CHANGES.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+Change Log
+==========
+
+v0.0.12
+-------
+
+Highlights
+''''''''''
+
+Example scripts are now availible for download as IPython Notebooks
```
Comment (GaelVaroquaux) [inline CHANGES.rst]: Typo at "Availible"

File: sphinx_gallery/__init__.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 """
 import os
-__version__ = '0.0.11'
+__version__ = '0.0.12'
```
Comment (GaelVaroquaux) [inline sphinx_gallery/__init__.py]: I think that we could be terribly bold, and release 0.1.0

**Comment by Titan-C** [comment]
Aggressively updating to version v0.1.0

File: CHANGES.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Highlights
+''''''''''
+
+Example scripts are now available for download as IPython Notebooks
+`#75 <https://github.com/sphinx-gallery/sphinx-gallery/pull/75>`_
+
+New features
+------------
+
+* Configurable filename pattern to which example scripts are executed
```
Comment (lesteve) [inline CHANGES.rst]: "to select which" rather than "to which"

File: doc/getting_started.rst [hunk 4]
```diff
@@ -1 +1,23 @@
 .. include:: ../README.rst
+
```
Comment (lesteve) [inline doc/getting_started.rst]: Is this supposed to be part of this PR?

Comment (Titan-C) [inline doc/getting_started.rst]: I'm adding this extra information to close #90, since Sphinx -Gallery has its namespace

**Pull request merged**